config/config.yaml

```diff
@@ -172,6 +171,17 @@ analysis:
     linkage: 'ward'
     # 'euclidean', 'manhattan', 'cosine'
     metric: 'euclidean'
+    # adds kernel density to pca plots
+    # default metric used is euclidean for kernel_density
+    kernel_density: true
+    # hyperparameter to control the smoothness of estimated kernel density
+    # single float, 'scott', 'silverman'
+    bandwidth: 1.0
+    # hyperparameter kernel to use for kernel_density
+    # 'gaussian', 'tophat', 'epanechnikov', 'exponential', 'linear', 'cosine'
+    kernel: 'gaussian'
```

Comment:
I noted this in a comment elsewhere already. If we don't save the PCA plots, I'm not sure how a user will know how to tune these, which may imply we don't need to expose them.